src/templates/data/database/postgres/migration.ejs

```diff
@@ -1,9 +1,13 @@
 
 exports.up = async function (knex) {
-    return knex.schema
-    .createTableIfNotExists('<%- props.table %>', function (table) {
-        table.increments('id').primary()
-        <%- props.columns %>
+    knex.schema.hasTable('<%- props.table %>').then(function(exists) {
+        if(!exists) {
```

Comment:
to avoid many indentations level, we can invert the if logic like
``` js
if(exists) return 
return knex.schema
            .createTable('<%- props.table %>', function (table) {
[...]

Reply:
done